### PR TITLE
tests for QCI-93

### PR DESCRIPTION
**Pull request by apagac**
Super easy tests using already defined fixtures. Please comment on the names of the test files, their location, etc.

### CODE REVIEW

File: tests/test_deployments_button_QCI93.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#QCI-93
```
Comment (jamesooden) [inline tests/test_deployments_button_QCI93.py]: Could we add some text to explain what it is we are testing.   I realize we have a reference to a Polarion ID but someday we won't use Polarion ande that reference may or may not be meaningful.

I would also provide a full link to the Polarian test case.

File: tests/test_new_deployment_button_QCI93.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#QCI-93
```
Comment (jamesooden) [inline tests/test_new_deployment_button_QCI93.py]: Same comment as in the first test.

**Comment by jamesooden** [comment]
:+1:

**Comment by apagac** [comment]
Added direct links to polarion testcase; added description of tests according to polarion test steps; improved test according to polarion test steps

**Comment by LandoCalrizzian** [comment]
As a public repo, I don't think we need to reference internal resources (polarion testcase number and url) in the pytests.  The fusor-qe test should be descriptive of what QCI element it's testing but they shouldn't reference polarion test cases directly. If we change the polarion testcase number, link, requirements,... that would also trigger an update in every fusor-qe testcase referencing it.

If there is a need to link polarion to fusor-qe tests then there are other ways to create a master polarion testclass that would link a testcase number to a specific pytest.

**Comment by jamesooden** [comment]
Yeah, as I think about it you are right Landon.   I think the thing we should do is put a link to the test code in the test case.   That keeps it internal stuff pointing to external stuff.

**Comment by apagac** [comment]
Removed link to polarion, left the ID of the testcase. Linking the test with the polarion testcase will be done from the polarion side as soon as this gets merged.

**Comment by LandoCalrizzian** [comment]
Looks good

**Pull request merged**